exporter.go

```diff
@@ -377,7 +378,10 @@ func (e *Exporter) createSliceLayer(appDir, layerID string, files []string) (Sli
 	// the directories and delete if empty as a result of previous removal
 	var dirs []string
 	for file := range fileSet {
-		stat, _ := os.Stat(file)
+		stat, err := os.Stat(file)
+		if err != nil {
+			return SliceLayer{}, errors.Wrap(err, "failed to stat file")
```

Comment:
@ekcasey I'm guessing these should follow the existing pattern and not be fatal too. I'll adjust (and revert the two below).